packages/storage/src/core/uploaders/ipfs-uploader.ts

```diff
@@ -308,8 +353,11 @@ export class IpfsUploader implements IStorageUploader<IpfsUploadBatchOptions> {
         return reject(new Error("Unknown upload error occured"));
       });
 
-      xhr.open("POST", PINATA_IPFS_URL);
-      xhr.setRequestHeader("Authorization", `Bearer ${token}`);
+      xhr.open("POST", ipfsUploadUrl);
+
+      if (uploadToken) {
+        xhr.setRequestHeader("Authorization", `Bearer ${uploadToken}`);
```

Comment:
@nessup do you think we should be passing the default API key here in preparation?